### ISSUE TITLE
Expose Config

### ISSUE THREAD

**Issue by sporto** [tripokey/elm-fuzzy#3]
The lib has functions that return a `Config` e.g.

```
addPenalty : Int -> Config
```

This means that I can get a config and pass it around in my app, which I'm doing here https://github.com/sporto/elm-select/blob/master/src/Select/Utils.elm#L28

But I have no way to add a type signature to my functions as the lib doesn't expose Config. If I try to add a type signature the Elm compiler will complain that I cannot use `Config`.

So it would be good to expose `Config` for completion.

**Comment by sporto** [comment]
If you don't want to expose this union type you could have something like:

```
type PrivateConfig
    = AddPenalty Int
    | RemovePenalty Int
    | MovePenalty Int
    
type Config = Config PrivateConfig
```

**Comment by tripokey** [comment]
Fixed by https://github.com/tripokey/elm-fuzzy/commit/31d0ad97e87603038b8bae408a4d0e42c1b14384

and released in 5.1.0.

elm-select looks nice btw :-)

**Comment by sporto** [comment]
Thanks
